jsapp/js/projects/myProjectsRoute.tsx

```diff
@@ -84,6 +127,50 @@ function MyProjectsRoute() {
       </UsageContext.Provider>
 
       <section className={styles.root}>
+        {invite.status && (
+          <div className={banner ? styles.banner : styles.noBanner}>
```

Comment:
Would `invite.status && banner &&` be sufficient? Maybe we don't need that `noBanner` classname?